salt/metalk8s/containerd/installed.sls

```diff
@@ -6,7 +6,7 @@ include:
 
 Install container-selinux:
   pkg.installed:
-    - name: container-selinux
+    - source: ftp://ftp.pbone.net/mirror/ftp.scientificlinux.org/linux/scientific/7x/external_products/extras/x86_64/container-selinux-2.77-1.el7_6.noarch.rpm
```

Comment:
This doesn't work for me.
I had to modify like this : 
```
Install container-selinux:
  pkg.installed:
    - sources:
      - container-selinux: ftp://ftp.pbone.net/mirror/ftp.scientificlinux.org/linux/scientific/7x/external_products/extras/x86_64/container-selinux-2.77-1.el7_6.noarch.rpm

```

Reply:
That's odd, 'worked on my machine'.
Anyway, given the fix, could you squash commits instead?